packages/angular/src/flyout-menu/flyout-menu-pane.component.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Optional,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  AnimationFrameService,
+  CloseMeta,
+  closestAttr,
+  Dialog,
+  ElementService,
+  position,
+} from 'carbon-components-angular';
+import { I18n } from 'carbon-components-angular/i18n';
+
+/**
+ * The Filter menu component encapsulates the OverFlowMenu directive, and the flyout iconography
+ * into one convienent component
+ *
+ * [See demo](../../?path=/story/components-flyout-menu--basic)
+ *
+ * html:
+ * ```
+ * <ai-flyout-menu-pane>
+ *	options
+ * </ai-flyout-menu-pane>
+ * ```
+ *
+ * <example-url>../../iframe.html?id=components-flyout-menu--basic</example-url>
+ */
+@Component({
+  selector: 'ai-flyout-menu-pane',
+  template: `
+    <div
+      #dialog
+      [id]="dialogConfig.compID"
+      [attr.role]="role"
+      [attr.data-floating-menu-direction]="dialogConfig.placement"
+      class="bx--tooltip bx--tooltip--shown iot--flyout-menu--body"
+      [ngClass]="{
+        'iot--flyout-menu--body__bottom-start': position === 'bottom-start',
+        'iot--flyout-menu--body__bottom-end': position === 'bottom-end',
+        'iot--flyout-menu--body__top-start': position === 'top-start',
+        'iot--flyout-menu--body__top-end': position === 'top-end',
+        'iot--flyout-menu--body__left-start': position === 'left-start',
+        'iot--flyout-menu--body__left-end': position === 'left-end',
+        'iot--flyout-menu--body__right-start': position === 'right-start',
+        'iot--flyout-menu--body__right-end': position === 'right-end',
```

Comment:
Do we need all these? Or is this something our dialog component can handle already?

Reply:
These are iot prefix so I don't think dialog would handle them.

Reply:
Right, but what styles do they actually apply? If it's just some `left`/`right` we can probably skip them, if it has more specific style hooks (content alignment or something?) then we'll have to keep them. As-is if it's just `left`/`right` I'm guessing the dialog is overriding it with inline styles

Reply:
they do more than that 👍